.github/workflows/deploy-generic.yml

```diff
@@ -0,0 +1,100 @@
+name: Deploy
+
+on:
+  workflow_call:
+    inputs:
+      env:
+        description: "which environment to deploy to"
+        required: true
+        type: string
+      ecr_repository:
+        description: "ecr repo to hold container image"
+        required: true
+        type: string
+      ecr_region:
+        description: "ecr region to hold container image"
+        required: true
+        type: string
+      debug:
+        description: "flag for django's debug mode"
+        required: true
+        type: string
+      django_allowed_hosts:
+        description: "django allowed hosts"
+        required: true
+        type: string
+      catalogue_url:
+        description: "url for backend datahub catalogue"
+        required: true
+        type: string
```

Comment:
```suggestion
```
Not needed, see [my other comment in the test workflow](https://github.com/ministryofjustice/find-moj-data/pull/132/files#r1517855557).